src/main/java/frc/team2412/robot/subsystem/DrivebaseSubsystem.java

```diff
@@ -248,6 +287,18 @@ private void updateModules(HolonomicDriveSignal driveSignal, double dt) {
         }
     }
 
+    public void updateModules(ChassisSpeeds chassisSpeeds) {
+        HolonomicDriveSignal holonomicDriveSignal = new HolonomicDriveSignal(
+                new Vector2(
+                        (chassisSpeeds.vxMetersPerSecond / moduleMaxVelocityMetersPerSec),
+                        (chassisSpeeds.vyMetersPerSecond / moduleMaxVelocityMetersPerSec)),
```

Comment:
@stewluj - this update to scale by 'max velocity per second' was the one thing missing that made it all work.